pkg/events/derive/symbols_loaded.go

```diff
@@ -75,7 +78,12 @@ func (symbsLoadedGen *symbolsLoadedEventGenerator) deriveArgs(event trace.Event)
 
 	soSyms, err := symbsLoadedGen.soLoader.GetExportedSymbols(loadingObjectInfo)
 	if err != nil {
-		return nil, err
+		// This error happens frequently in some environments, so we need to silence it to reduce spam.
+		if symbsLoadedGen.isDebug {
```

Comment:
https://github.com/aquasecurity/tracee/issues/2036#issuecomment-1211524984

Reply:
Now that we have the error counting method, can we remove the debug mode check here?

Reply:
I thought we would give the error in debug mode only (which I think it is the correct thing).

Reply:
IIRC we report runtime errors to stderr when they occur without checking for debug mode, exception are sometimes made for initialization code errors (for example setting up the runtime enrichers, if a runtime is "missing" will emit a false positive error).
It may be better in the future that errors will only be emitted if debug mode is set (better defined through log levels), but we should normalize that in one go, not break standards in the middle. We can, and should, do that alongside the logger PR.

Since the code won't spam errors now, I think it's safe to remove the debug condition.

Reply:
https://github.com/aquasecurity/tracee/issues/2036#issuecomment-1212033996